### PR TITLE
ref(js): Remove deprecatedforms from Auth views

### PR DESCRIPTION
**Pull request by evanpurkhiser**
**[!!] These views are NOT used in production!**

Just removes some deprecatedforms field usage. This is nearly the last
uasages of these fields, we're close to deleting them

### CODE REVIEW

File: static/app/views/auth/loginForm.tsx [hunk 1]
```diff
@@ -58,101 +56,71 @@ const LoginProviders = ({
...
+          // the application state after user login.
+
+          ConfigStore.set('user', response.user);
+
+          // TODO(epurkhiser): Reconfigure sentry SDK identity
+
+          browserHistory.push({pathname: response.nextUri});
+        }}
+        onSubmitError={response => {
+          // TODO(epurkhiser): Need much better error handling here
```
Comment (priscilawebdev) [inline static/app/views/auth/loginForm.tsx:85]: agree

File: static/app/views/auth/registerForm.tsx [hunk 2]
```diff
@@ -1,135 +1,94 @@
...
+            ['true', t('Yes, I would like to receive updates via email')],
+            ['flase', t("No, I'd prefer not to receive these updates")],
+          ]}
+          help={tct(
+            `We'd love to keep you updated via email with product and feature
+           announcements, promotions, educational materials, and events. Our
+           updates focus on relevant information, and we'll never sell your data
+           to third parties. See our [link] for more details.`,
+            {
+              link: <a href="https://sentry.io/privacy/">Privacy Policy</a>,
```
Comment (priscilawebdev) [inline static/app/views/auth/registerForm.tsx:83]: shall we use `ExternalLink` here?

**Review by priscilawebdev — APPROVED** [review]
lgtm... Goo team!!

File: static/app/views/auth/registerForm.tsx (hunk 2 above)
Comment (evanpurkhiser) [reply static/app/views/auth/registerForm.tsx:83]: ah yea, I can change

**Pull request merged**